ethereum_history_api/tests/package.json

```diff
@@ -4,6 +4,7 @@
   "type": "module",
   "scripts": {
     "test:e2e": "vitest run --config vitest.e2e.config.ts",
+    "test:e2e:coverage": "vitest run --config vitest.e2e.config.ts --coverage && echo '\nTo open a nicer version of this coverage report, run `open coverage/index.html`\n'",
```

Comment:
It doesn't make semantic sense to have coverage report in e2e tests as we cover Noir and Solidity code in those tests and all ts files are just helpers. Let's remove